its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SolutionKindTest.java

```diff
@@ -47,11 +51,14 @@ class SolutionKindTest {
   private static final String SONAR_RULES_PREFIX = "csharpsquid:";
 
   @Test
+  @EnabledOnOs(OS.WINDOWS) // Xamarin.IOS is not supported on unix, https://learn.microsoft.com/en-us/answers/questions/321541/is-it-possible-to-build-and-deploy-xamarin-ios-pro
```

Comment:
This is a bit nitpick but it can mess up with the formatting of IntelliJ:
```suggestion
  // Xamarin.IOS is not supported on unix, https://learn.microsoft.com/en-us/answers/questions/321541/is-it-possible-to-build-and-deploy-xamarin-ios-pro
  @EnabledOnOs(OS.WINDOWS)
```